tests/datasource/data_connector/test_sql_data_connector.py

```diff
@@ -74,20 +74,22 @@ def test_basic_self_check(test_cases_for_sql_data_connector_sqlite_execution_eng
     }
 
 
+@pytest.mark.parametrize("splitter_method_name_prefix", ["_", ""])
 def test_get_batch_definition_list_from_batch_request(
+    splitter_method_name_prefix,
     test_cases_for_sql_data_connector_sqlite_execution_engine,
 ):
     random.seed(0)
     db = test_cases_for_sql_data_connector_sqlite_execution_engine
 
     config = yaml.load(
-        """
+        f"""
     name: my_sql_data_connector
     datasource_name: FAKE_Datasource_NAME
 
     assets:
         table_partitioned_by_date_column__A:
-            splitter_method: _split_on_column_value
+            splitter_method: {splitter_method_name_prefix}split_on_column_value
```

Comment:
super interesting and elegant way to deal with this @anthonyburdi . Love what you're doing with introducing parameterized tests.